Taquito build now requires pre-installation libusb-1.0 and libudev-dev

when you `npm run bootstrap` taquito after fresh installation there is an error that libusb-1.0 is missing.

this can be fixed by 

sudo apt-get install libudev-dev && sudo apt-get install libusb-1.0-0-dev

- [ ] but we should instead document that as a requirement to build it locally

I had the same issue with a fresh installation.
Solved it by running: `sudo apt-get install libudev-dev`

Here is the error:
```
npm ERR! code 1
npm ERR! path /home/roxane/src/ecadLabs/taquito/integration-tests/node_modules/usb
npm ERR! command failed
npm ERR! command sh -c prebuild-install --runtime napi --target 4 --verbose || node-gyp rebuild
npm ERR! make: Entering directory '/home/roxane/src/ecadLabs/taquito/integration-tests/node_modules/usb/build'
npm ERR!   CC(target) Release/obj.target/libusb/libusb/libusb/core.o
npm ERR!   CC(target) Release/obj.target/libusb/libusb/libusb/descriptor.o
npm ERR!   CC(target) Release/obj.target/libusb/libusb/libusb/hotplug.o
npm ERR!   CC(target) Release/obj.target/libusb/libusb/libusb/io.o
npm ERR!   CC(target) Release/obj.target/libusb/libusb/libusb/strerror.o
npm ERR!   CC(target) Release/obj.target/libusb/libusb/libusb/sync.o
npm ERR!   CC(target) Release/obj.target/libusb/libusb/libusb/os/poll_posix.o
npm ERR!   CC(target) Release/obj.target/libusb/libusb/libusb/os/threads_posix.o
npm ERR!   CC(target) Release/obj.target/libusb/libusb/libusb/os/linux_usbfs.o
npm ERR!   CC(target) Release/obj.target/libusb/libusb/libusb/os/linux_udev.o
npm ERR! make: Leaving directory '/home/roxane/src/ecadLabs/taquito/integration-tests/node_modules/usb/build'
npm ERR! prebuild-install info begin Prebuild-install version 5.3.6
npm ERR! prebuild-install info install installing standalone, skipping download.
npm ERR! gyp info it worked if it ends with ok
npm ERR! gyp info using node-gyp@5.1.1
npm ERR! gyp info using node@16.13.0 | linux | x64
npm ERR! gyp info find Python using Python version 2.7.18 found at "/usr/bin/python2"
npm ERR! gyp http GET https://nodejs.org/download/release/v16.13.0/node-v16.13.0-headers.tar.gz
npm ERR! gyp http 200 https://nodejs.org/download/release/v16.13.0/node-v16.13.0-headers.tar.gz
npm ERR! gyp http GET https://nodejs.org/download/release/v16.13.0/SHASUMS256.txt
npm ERR! gyp http 200 https://nodejs.org/download/release/v16.13.0/SHASUMS256.txt
npm ERR! (node:111739) [DEP0150] DeprecationWarning: Setting process.config is deprecated. In the future the property will be read-only.
npm ERR! (Use `node --trace-deprecation ...` to show where the warning was created)
npm ERR! gyp info spawn /usr/bin/python2
npm ERR! gyp info spawn args [
npm ERR! gyp info spawn args   '/home/roxane/src/ecadLabs/taquito/node_modules/node-gyp/gyp/gyp_main.py',
npm ERR! gyp info spawn args   'binding.gyp',
npm ERR! gyp info spawn args   '-f',
npm ERR! gyp info spawn args   'make',
npm ERR! gyp info spawn args   '-I',
npm ERR! gyp info spawn args   '/home/roxane/src/ecadLabs/taquito/integration-tests/node_modules/usb/build/config.gypi',
npm ERR! gyp info spawn args   '-I',
npm ERR! gyp info spawn args   '/home/roxane/src/ecadLabs/taquito/node_modules/node-gyp/addon.gypi',
npm ERR! gyp info spawn args   '-I',
npm ERR! gyp info spawn args   '/home/roxane/.cache/node-gyp/16.13.0/include/node/common.gypi',
npm ERR! gyp info spawn args   '-Dlibrary=shared_library',
npm ERR! gyp info spawn args   '-Dvisibility=default',
npm ERR! gyp info spawn args   '-Dnode_root_dir=/home/roxane/.cache/node-gyp/16.13.0',
npm ERR! gyp info spawn args   '-Dnode_gyp_dir=/home/roxane/src/ecadLabs/taquito/node_modules/node-gyp',
npm ERR! gyp info spawn args   '-Dnode_lib_file=/home/roxane/.cache/node-gyp/16.13.0/<(target_arch)/node.lib',
npm ERR! gyp info spawn args   '-Dmodule_root_dir=/home/roxane/src/ecadLabs/taquito/integration-tests/node_modules/usb',
npm ERR! gyp info spawn args   '-Dnode_engine=v8',
npm ERR! gyp info spawn args   '--depth=.',
npm ERR! gyp info spawn args   '--no-parallel',
npm ERR! gyp info spawn args   '--generator-output',
npm ERR! gyp info spawn args   'build',
npm ERR! gyp info spawn args   '-Goutput_dir=.'
npm ERR! gyp info spawn args ]
npm ERR! gyp info spawn make
npm ERR! gyp info spawn args [ 'BUILDTYPE=Release', '-C', 'build' ]
npm ERR! ../libusb/libusb/os/linux_udev.c:40:10: fatal error: libudev.h: No such file or directory
npm ERR!    40 | #include <libudev.h>
npm ERR!       |          ^~~~~~~~~~~
npm ERR! compilation terminated.
npm ERR! make: *** [libusb.target.mk:152: Release/obj.target/libusb/libusb/libusb/os/linux_udev.o] Error 1
npm ERR! gyp ERR! build error 
npm ERR! gyp ERR! stack Error: `make` failed with exit code: 2
npm ERR! gyp ERR! stack     at ChildProcess.onExit (/home/roxane/src/ecadLabs/taquito/node_modules/node-gyp/lib/build.js:194:23)
npm ERR! gyp ERR! stack     at ChildProcess.emit (node:events:390:28)
npm ERR! gyp ERR! stack     at Process.ChildProcess._handle.onexit (node:internal/child_process:290:12)
npm ERR! gyp ERR! System Linux 5.11.0-27-generic
npm ERR! gyp ERR! command "/home/roxane/.nvm/versions/node/v16.13.0/bin/node" "/home/roxane/src/ecadLabs/taquito/node_modules/.bin/node-gyp" "rebuild"
npm ERR! gyp ERR! cwd /home/roxane/src/ecadLabs/taquito/integration-tests/node_modules/usb
npm ERR! gyp ERR! node -v v16.13.0
npm ERR! gyp ERR! node-gyp -v v5.1.1
npm ERR! gyp ERR! not ok
```

--- a/README.md
+++ b/README.md
@@ -117,6 +117,13 @@ __Make sure you have a version of Node.js supported by Taquito__
 * Run `lerna bootstrap --force-local`
 * Run `npm run build`
 
+---
+**NOTE**
+If you are getting the following error on a fresh install: `fatal error: libudev.h: No such file or directory`;
+try installing the package libudev-dev.
+Example for Ubuntu: `sudo apt-get install libudev-dev`
+
+---
 
 ### Useful npm command targets
 

